Sources/joyfill-components-swift/Components/Table/ViewTable.swift

```diff
@@ -65,12 +83,16 @@ public class ViewTable: UIViewController, TextFieldCellDelegate, TextViewCellDel
         selectedIndexPath = nil
     }
     
+    public func updateTitle(text: String) {
+        tableFloorsLabel.labelText = text
```

Comment:
So this is odd that the name is `tableFloorsLabel`? That seems to refer back to the names in the Figma designs but that stuff should not be taken literally as the TableField is dynamic and will receive all sorts of data. Code should not be named after dynamic natures of the designs

Reply:
Done